### PR TITLE
Added Junit Tests and Logging 

### PR DESCRIPTION
**Pull request by leonlowzd**
1. Added JunitTest for data packages
2. Added Logging for ON/Off Command

### CODE REVIEW

**Comment by leonlowzd** [comment]
Addresses #209

**Review by zongxian-ctrl — COMMENTED** [review]
Think you can add more logging with Level.WARNING if fail to turn on or off the appliance?

**Comment by leonlowzd** [comment]
> Think you can add more logging with Level.WARNING if fail to turn on or off the appliance?

You are right, I should add more for warning.

File: src/test/java/seedu/smarthomebot/logic/commands/OffCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        assertTrue(applianceList.getAppliance(0).switchOff());
+    }
+
+    @Test
+    void switchOff_Appliance_switchOffPreviously() {
+        assertFalse(applianceList.getAppliance(0).switchOff());
+    }
+}
+
+
```
Comment (Ang-Cheng-Jun) [inline src/test/java/seedu/smarthomebot/logic/commands/OffCommandTest.java:51]: Maybe can do the catch Exception for this part. You can see how i did for AddCommand

File: src/test/java/seedu/smarthomebot/logic/commands/OnCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        assertTrue(applianceList.getAppliance(0).switchOn());
+    }
+
+    @Test
+    void switchOn_Appliance_switchOnPreviously() {
+        applianceList.getAppliance(0).switchOn();
+        assertFalse(applianceList.getAppliance(0).switchOn());
+    }
+}
+
```
Comment (Ang-Cheng-Jun) [inline src/test/java/seedu/smarthomebot/logic/commands/OnCommandTest.java:47]: Maybe can do the catch Exception for this part. You can see how i did for AddCommand

**Review by Ang-Cheng-Jun — COMMENTED** [review]
Overall, well done

File: src/test/java/seedu/smarthomebot/logic/commands/OnCommandTest.java (hunk 2 above)
Comment (leonlowzd) [reply src/test/java/seedu/smarthomebot/logic/commands/OnCommandTest.java:47]: Sure, I can add more tests on that as well.

**Pull request merged**